doc/building_blocks/data_preparation.rst

```diff
@@ -99,13 +101,14 @@ a look at it.
     :start-after: # Simple mask extraction from EPI images
     :end-before: # Generate mask with strong opening
 
-____
 
-.. figure:: ../auto_examples/manipulating_visualizing/images/plot_mask_computation_3.png
+.. figure:: ../auto_examples/manipulating_visualizing/images/plot_mask_computation_2.png
     :target: ../auto_examples/plot_mask_computation.html
-    :align: right
+    :align: left
```

Comment:
I think you can just remove the :align: lines (i.e. it defaults to left by default)